### ISSUE TITLE
[ListView] Content of other component disappear when I scroll down

### ISSUE THREAD

**Issue by agrass** [facebook/react-native#7595]
Hi!

 I have a problem with the scroll of a ListView. First I render a ListView  and then I unmount the ListView to mount a Picker. The problem is when I change the scroll position on the list view and then mount the Picker, this new component is not showing. Then when I move the scroll on this new component it appears, its wierd.

I tried to do ```this._list.scrollTo({y: 0})```  in the willUnmount of the component that contains the ListView but it didn't work.

I'm working with Android and RN 0.24.

![example](https://docs.google.com/uc?id=0BwV8rTO_u_siemlFcGdmVzJXbDQ&export=download)

Anyone have any idea of how to solve this?

Thanks!

**Comment by agrass** [comment]
Fixed using ```this._scrollView.scrollTo({y: 0})``` in a parent ScrollView containing both elements. Anyway still wierd that if the content size changed the ScrollView didn't automatically notice that and update the scroll Y position to a valid position considering the new size.

**Comment by DanielMSchmidt** [comment]
@facebook-github-bot answered

**Comment by facebook-github-bot-0** [comment]
Closing this issue as @DanielMSchmidt says the question asked has been answered. Please help us by asking questions on [StackOverflow](http://stackoverflow.com/questions/tagged/react-native). StackOverflow is amazing for Q&A: it has a reputation system, voting, the ability to mark a question as answered. Because of the reputation system it is likely the community will see and answer your question there. This also helps us use the GitHub bug tracker for bugs only.

**Comment by agrass** [comment]
@DanielMSchmidt  It's answered but I think is still a bug that the ScrollView didn't update the "y" position when content changed (to a valid "y" position at least).

**Comment by dbonner1987** [comment]
I have also came across this issue – not sure it should be closed.

To recreate a similar issue, populate a ListView with posts for example. onPress and navigate to one of these posts, alter it, save it and hit the back button. Any ListView.dataSource change causes a blank screen like the example above, requiring the user to drag the screen to make the ListView visible.

The only solution I have found so far matches what @agrass has said. Wrap the ListView in a ScrollView (not ideal) and when the dataSource needs changed, fire `this.refs.listView.scrollTo({y: 0});`.
